metafacture-csv/src/test/java/org/metafacture/csv/CsvEncoderTest.java

```diff
@@ -0,0 +1,179 @@
+/*
+ *  Copyright 2018-2023 Deutsche Nationalbibliothek et al
+ *
+ *  Licensed under the Apache License, Version 2.0 the "License";
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.metafacture.csv;
+
+import org.metafacture.framework.ObjectReceiver;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.exceptions.base.MockitoAssertionError;
+
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+/**
+ * Tests for {@link CsvEncoder}.
+ *
+ * @author eberhardtj (j.eberhardt@dnb.de)
+ * @author Pascal Christoph (dr0i)
+ * @author Jens Wille
+ */
+public final class CsvEncoderTest {
+
+    @Mock
+    private ObjectReceiver<String> receiver;
+    private static final String LITERAL1 = "column 1";
+    private static final String LITERAL2 = "column 2";
+    private static final String RECORD_ID1 = "1";
+    private static final String RECORD_ID2 = "2";
+    private static final String RECORD_ID3 = "3";
+    private static final String VALUE1 = "a";
+    private static final String VALUE2 = "b";
+    private static final String VALUE3 = "c";
+    private static final String VALUE4 = "d";
+    private static final String VALUE5 = "e";
+    private static final String VALUE6 = "f";
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
This line _could_ be replaced with a `MockitoRule`.

Reply:
thx, did this in [67b4412](https://github.com/metafacture/metafacture-core/pull/486/commits/67b44124e1a7a3ed16b25ab2aac85a30a0234b49).

Reply:
Great, but now you got unused imports: `org.junit.Before` and `org.mockito.MockitoAnnotations`.

Reply:
Thx for improving.